tasks/quality.js

```diff
@@ -111,6 +114,25 @@ module.exports = function(grunt) {
     analyze.push('phpmd:custom');
   }
 
+  if (grunt.config.get('config.eslint') != undefined) {
+    var eslintConfig = grunt.config.get('config.eslint'),
+      eslintTarget = eslintConfig.srcPaths || [
+          '<%= config.srcPaths.drupal %>/**/*.js',
+          '!<%= config.srcPaths.drupal %>/sites/**/files/**/*.js'
+        ],
+      eslintConfigFile = eslintConfig.configFile || './.eslintrc',
+      eslintIgnoreError = eslintConfig.hasOwnProperty('ignoreError') ? eslintConfig.ignoreError : true,
+      eslintName = eslintIgnoreError ? 'force:eslint' : 'eslint';
+    grunt.config('eslint', {
+      options: {
+        configFile: eslintConfigFile
+      },
+      target: eslintTarget
+    });
+    validate.push(eslintName);
+    analyze.push(eslintName);
```

Comment:
Right now everything in the analyze task is outputting to a file for Jenkins/etc to process. grunt-eslint/eslint does support the `checkstyle` format that phpcs uses.